pages/App.tsx

```diff
@@ -14,7 +14,7 @@ const games: string[] = [
 
 const languages: string[] = ["English", "German", "Spanish", "Japanese"];
 
-function App() {
+export default function App() {
```

Comment:
This is not page should not be here. https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts

Reply:
done